rfcs/013-release_deployment_tracking/v1/README.md

```diff
@@ -0,0 +1,181 @@
+# Version 1 
+
+## Implementation from 24/01/2019
+
+This solution describes a set of steps to track build, release and deployment but is deliberately agnostic of deployment mechanism. 
```

Comment:
nit: "deployment" ~> "deploy"
nit: are you missing a comma between "track" and "build"?

What are we building/releasing/deploying?